src/sass/myservice-sliding-panel-top.scss

```diff
@@ -190,7 +190,7 @@
     }
 
     &::after {
-      background-image: uikit-svguri('<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"\a   viewBox="0 0 282.4 77.7" style="enable-background:new 0 0 282.4 77.7;" xml:space="preserve">\a<style type="text/css">\a  .st0{fill:#FFFFFF;}\a  .st1{fill:#D7EA6A;}\a</style>\a<g>\a  <path class="st0" d="M0,60h18.7V38.3c0-3.5,0.6-6,1.9-7.5c1.3-1.5,2.9-2.3,5-2.3c1.2,0,2.3,0.4,3.3,1.1c0.9,0.7,1.6,1.7,1.9,2.8\a    c0.2,0.7,0.3,2.1,0.3,4.1v23.6h18.7v-22c0-3.3,0.6-5.7,1.9-7.2c1.3-1.5,2.9-2.3,5-2.3c1.7,0,3.2,0.8,4.4,2.4c0.8,1,1.2,2.6,1.2,4.6\a    V60H81V31.5c0-6.1-1.4-10.7-4.3-13.5c-2.9-2.9-6.8-4.4-11.9-4.4c-3.5,0-6.4,0.5-8.7,1.6c-2.4,1.1-5,3.1-7.8,6\a    c-1.4-2.5-3.2-4.4-5.4-5.7c-2.2-1.3-5.2-1.9-9-1.9c-3.5,0-6.5,0.6-9,1.7c-2.5,1.2-5,3.1-7.4,5.8V15H0V60z"/>\a  <path class="st0" d="M93,63.2c-0.7,0.6-1.3,0.8-3.4,0.8c-1.8,0-2.1-0.3-4.6-1.2l-0.1,13.9c5.2,0.7,6.6,1,9.1,1\a    c6.2,0,10.4-1.1,13.5-3.9c2.1-2,4.3-7.1,6.3-12.7l15.4-46h-18.9l-6.5,29l-6.3-29H77.8l16.4,45.3C94.2,60.4,94.3,62.2,93,63.2z"/>\a</g>\a<g>\a  <path class="st1" d="M193.2,55.2c4.7,3.9,10.8,5.8,18.3,5.8c8.5,0,15.1-2.3,19.9-7s7.2-10.7,7.2-18c0-6.5-1.9-12-5.8-16.5\a    c-4.8-5.5-11.9-8.9-21.3-8.9c-8.2,0-14.8,2.9-19.6,7.6s-7.2,10.7-7.2,18C184.7,44.1,187.5,50.4,193.2,55.2z M205.7,26.6\a    c1.6-1.9,3.6-2.9,6-2.9c2.3,0,4.2,1,5.7,2.8c1.5,1.9,2.3,4.7,2.3,8.5c0,4.1-0.8,7.1-2.3,8.9c-1.5,1.9-3.5,3.3-5.9,3.3\a    s-4.4-1.5-5.9-3.3c-1.6-1.9-2.3-4.8-2.3-8.7C203.4,31.4,204.2,28.5,205.7,26.6z"/>\a  <polygon class="st1" points="266,58.8 282.4,12.4 262.1,12.4 256.1,37.6 250.4,12.4 230.7,12.4 246.6,58.8 	"/>\a  <path class="st1" d="M168.2,39.3v4.8c-2.3,1.2-4.3,2.1-6,2.6s-3.6,0.8-5.5,0.8c-4.1,0-7.2-1.3-9.5-4c-2.3-2.6-3.4-7-3.4-13.2\a    c0-5.8,1.2-10.1,3.5-12.7c2.3-2.7,5.5-4,9.4-4c2.6,0,4.8,0.5,6.5,1.4s2.9,2.3,3.6,4l17.6-2.5c-1.1-3.8-2.6-6.9-4.7-9.3\a    c-2.1-2.5-4.7-4.3-7.9-5.4c-3.2-1.2-8-1.8-14.4-1.8c-6.6,0-11.9,1-15.8,2.9c-5,2.5-8.9,6.1-11.5,10.9s-3.9,10.4-3.9,16.8\a    c0,6.1,1.2,11.6,3.6,16.3c2.4,4.7,5.8,8.3,10.2,10.7c4.4,2.4,10,3.6,17,3.6c5.6,0,10.5-0.7,14.6-2.1s8.8-4,14.2-7.9V25.7h-28.4\a    v13.6H168.2z"/>\a</g>\a</svg>');
+      background-image: url("/docs/images/mygov-logo.svg");
```

Comment:
this might need to be a relative path, ../images/mygov-logo.svg

Reply:
ok - will look shortly. will check how it is used elsewhere.

Reply:
just checked and three existing pages link to this via that same absolute path. I tested it in IE and it worked ok. what would you like me to do? happy either way.

Reply:
i'm thinking it might only work in the prototype. When we port it across the myservice app, it might be the reason it's not loading